core-tests/src/test/scala/au/com/agiledigital/rest/security/BodyParsersSpec.scala

```diff
@@ -33,20 +40,105 @@ class BodyParsersSpec(implicit ev: ExecutionEnv) extends BaseSpec with DataTable
       )
     )
 
+    val expectedUnsafeObject = Json.obj(
+      "foo" -> Json.obj(
+        "bar" -> "<span>bar</span> baz",
+        "barUnsafe" -> "<img src=\"foo\"><span>bar</span><script>alert('baz');</script> baz"
+      ),
+      "fooUnsafe" -> Json.obj(
+        "bar" -> "<img src=\"foo\"><span>bar</span><script>alert('baz');</script> baz"
+      )
+    )
+
+    // @formatter:off
```

Comment:
Yes, we can, I will remove @formatter:off and configure Intellij to understand // format: OFF